### ISSUE TITLE
X stuck in fan

### ISSUE THREAD

**Issue by Jasonb11** [rmotr-curriculum/free-python-course-questions#36]
Seriously, what is the question here. I'm not sure what is needed to pass..

**Comment by Katwilliams12** [comment]
Agree... I don't remember talking about this. Also I basically copied the example for the test question and it's not working.
